backend/media/BUILD

```diff
@@ -0,0 +1,41 @@
+load("//tools/build:springboot.bzl", "springboot")
```

Comment:
`backend/media/resolver`? I understand why we didn't do yet (after all there's nothing else in the directory and probably nothing coming for a while). The proposed folder structure would create a certain symmetry. No strong opinion though, just though it was worth sharing

Reply:
It's a good point, but for now, I would keep it like so if only to keep it simple. I think it is harder to maintain that symmetry in core, e.g. look at the chat plugin which does not match the other sources. Or sources in general vs `api`. 

Overall I appreciate that we are a bit more flexible on core because it encourages us to think harder about directory structure.